website/docs/cli/commands/plan.mdx

```diff
@@ -290,6 +290,8 @@ The available options are:
   * 1 = Error
   * 2 = Succeeded with non-empty diff (changes present)
 
+- `-generate-config-out=PATH` - (Experimental) If `import` blocks are present in config, instructs Terraform to generate HCL for any imported resource that does not already have configuration. The config will be written to a file at PATH, which must not already exist. If the plan exits with an error, Terraform may still attempt to write config. After the plan has completed, review the generated config.
```

Comment:
[Suggestion to match your other PR](https://github.com/hashicorp/terraform/pull/33279/files):
```suggestion
- `-generate-config-out=PATH` - (Experimental) If import blocks are present in configuration, instructs Terraform to generate HCL for any imported resources not already present. The configuration is written to a new file at PATH, which must not already exist. Terraform may still attempt to write configuration if the plan errors.
```